aries-site/src/examples/templates/filtering/FilteringTable/FilteringTable.js

```diff
@@ -0,0 +1,132 @@
+import { useContext } from 'react';
+import { Box, DataTable, Heading, Menu, ResponsiveContext } from 'grommet';
+import {
+  StatusWarningSmall,
+  StatusCriticalSmall,
+  StatusGoodSmall,
+  StatusUnknownSmall,
+} from 'grommet-icons';
+
+import {
+  FilterControls,
+  FiltersProvider,
+  useFilters,
+} from '../../FilterControls';
+import { servers } from './mockData';
+
+export const FilteringTable = () => {
+  // Define which attributes should be made available for the user
+  // to filter upon
+  const filtersConfig = [
+    {
+      property: 'hardware.health.summary',
+      label: 'Status',
+      filterType: 'CheckBoxGroup',
+    },
+    {
+      property: 'state.connected',
+      label: 'State',
+      filterType: 'CheckBoxGroup',
+      render: datum => (datum ? 'Connected' : 'Not Connected'),
+    },
+    {
+      property: 'hardware.powerState',
+      label: 'Power',
+      filterType: 'CheckBoxGroup',
+    },
+    {
+      property: 'hardware.model',
+      label: 'Model',
+      filterType: 'CheckBoxGroup',
+    },
+  ];
+
+  return (
+    <Box pad="small" gap="medium">
+      <Heading level={2} margin="none">
+        Servers
+      </Heading>
+      <FiltersProvider>
+        <Box gap="medium">
+          <Box align="start" direction="row" justify="between" wrap>
+            <FilterControls
+              data={servers}
+              filters={filtersConfig}
+              primaryKey="id"
+              searchFilter={{ placeholder: 'Search servers...' }}
+            />
+            <Menu
+              kind="toolbar"
+              label="Actions"
+              items={[{ label: 'Power On' }, { label: 'Power Off' }]}
+            />
+          </Box>
+          <ServerResults />
+        </Box>
+      </FiltersProvider>
+    </Box>
+  );
+};
+
+const columns = [
+  {
+    property: 'hardware.serialNumber',
+    header: 'Serial',
+  },
+  {
+    property: 'state.connected',
+    header: 'State',
+    render: datum => (datum.state.connected ? 'Connected' : 'Not Connected'),
+  },
+  {
+    property: 'hardware.powerState',
+    header: 'Power',
+  },
+  {
+    property: 'hardware.model',
+    header: 'Model',
+  },
+];
+
+const statusIcons = {
+  Warning: <StatusWarningSmall color="status-warning" size="small" />,
+  OK: <StatusGoodSmall color="status-ok" size="small" />,
+  Critical: <StatusCriticalSmall color="status-critical" size="small" />,
+  Unknown: <StatusUnknownSmall color="status-unknown" size="small" />,
+};
+
+const ServerResults = () => {
+  const size = useContext(ResponsiveContext);
+  const { filteredResults, selected, setSelected } = useFilters();
+
+  return (
+    <Box height="medium" overflow="auto">
```

Comment:
When in dark, full-screen, the table checkboxes have different cell backgrounds. Need to specify background for uniformity.

![Screen Shot 2021-06-14 at 4 19 45 PM](https://user-images.githubusercontent.com/1756948/121967120-05d50100-cd2d-11eb-95ac-8d4dbbe0e65b.png)